### PR TITLE
Bybit :: fetchOrderBook update

### PR DESCRIPTION
**Pull request by carlosmiei**
- updated fetchOrderBook to handle all available market types

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -1639,7 +1639,7 @@ module.exports = class bybit extends Exchange {
         return this.parseTrades (result, market, since, limit);
     }
 
-    parseOrderBook (orderbook, symbol, timestamp = undefined, bidsKey = 'Buy', asksKey = 'Sell', priceKey = 'price', amountKey = 'size') {
+    parseCustomOrderBook (orderbook, symbol, timestamp = undefined, bidsKey = 'Buy', asksKey = 'Sell', priceKey = 'price', amountKey = 'size') {
```
Comment (kroitor) [inline js/bybit.js:1642]: Why do we rename it?

Comment (carlosmiei) [reply js/bybit.js:1642]: @kroitor Because for spot markets we are using the inherited `parseOrderBook`

File: js/bybit.js [hunk 2]
```diff
@@ -1680,16 +1713,36 @@ module.exports = class bybit extends Exchange {
...
+        //           {
+        //             "price": "5900.00000000",
+        //             "size": "0.9000",
+        //             "side": "Sell" // asks
+        //           }
+        //         ]
+        //    }
+        //
         const result = this.safeValue (response, 'result', []);
+        if (market['spot']) {
```
Comment (kroitor) [inline js/bybit.js:1740]: Let's still override parseOrderBook and do all these checks inside it, namely if it should be parsed with the parent method or with a custom code. We should still keep it all in `parseOrderBook`.

Comment (carlosmiei) [reply js/bybit.js]: @kroitor done

**Pull request merged**